README.md

```diff
@@ -51,3 +51,69 @@ at your option.
+| [ecb-demo](./ecb-demo/README.md)                      | Encryption        | AES electronic codebook mode encryption demo                          |
+| [gpiote-demo](./gpiote-demo/README.md)                | Digital Pins      | General-Purpose Input Output Tasks and Events module demo             |
+| [i2s-controller-demo](./i2s-controller-demo/README.md)| Audio             | Inter-IC Sound interface "controller mode (aka master mode)" demo     |
+| [i2s-peripheral-demo](./i2s-peripheral-demo/README.md)| Audio             | Inter-IC Sound interface "peripheral mode (aka slave mode)" demo      |
+| [lpcomp-demo](./lpcomp-demo/README.md)                | Analog Pins       | Low power voltage comparator demo                                     |
+| [ppi-demo](./ppi-demo/README.md)                      | Channels          | Programmable peripheral interconnect (PPI) demo                       |
+| [pwm-demo](./pwm-demo/README.md)                      | Digital Pins      | Pulse width modulation demo                                           |
+| [qdec-demo](./qdec-demo/README.md)                    | Sensor Decoding   | Quadrature sensor decoder (QDEC) demo                                 |
+| [rtic-demo](./rtic-demo/README.md)                    | Framework         | The Real-Time Interrupt-driven Concurrency framework demo             |
+| [spi-demo](./spi-demo/README.md)                      | Digital Pins      | Serial peripheral interface master (SPIM) with EasyDMA demo           |
+| [twi-ssd1306](./twi-ssd1306/README.md)                | Digital Pins      | I2C compatible Two-Wire Interface with the SSD1306 OLED Display demo  |
+| [twim-demo](./twim-demo/README.md)                    | Digital Pins      | I2C compatible Two-Wire Interface Master mode demo                    |
+| [twis-demo](./twis-demo/README.md)                    | Digital Pins      | I2C compatible Two-Wire Interface Slave mode demo                     |
+| [wdt-demo](./wdt-demo/README.md)                      | Timer             | Watchdog timer demo                                                   |
+
+
+## Running the Examples
+
+Each demo readme should contain instructions on how to run it. However, the information below describes the technologies used and can be used to troubleshoot your system setup. Run the demos from within their respective project directories. E.g. to run `ccm-demo`, you must be in the `nrf-hal/examples/ccm-demo/` directory and run `cargo run`.
+> Since the demos are stand-alone projects you would **NOT** typically run them with `cargo run --example xyz-demo` like some cargo projects are configured to do.
+
+There are many ways to setup programming and debugging with an embedded device. Here we will describe how to do this on the nRF52840-DK using the [probe-rs](https://probe.rs/) set of tools. 
+
+### Once Off System Setup
+
+Install the cross compilation toolchain to target the device.
+```console
+$ rustup target add thumbv7em-none-eabihf
```

Comment:
This is not the right target for all chips

Reply:
This a an example target. I have changed the wording above the command to give it more context.